packages/shorebird_cli/lib/src/shorebird_config_mixin.dart

```diff
@@ -6,6 +6,7 @@ import 'package:pubspec_parse/pubspec_parse.dart';
 import 'package:shorebird_cli/src/command.dart';
 import 'package:shorebird_cli/src/config/config.dart';
 import 'package:yaml/yaml.dart';
+import 'package:yaml_edit/yaml_edit.dart';
```

Comment:
I've used this package before, and I recall having a tickle in my brain about something being wrong with it. e.g. not maintaining comments or order or something. 🤷  Maybe whatever was wrong then is fixed now. :)